src/api/app/views/webui/project/responsive_ux/_actions.html.haml

```diff
@@ -0,0 +1,32 @@
+- content_for :actions do
+  - if (bugowners_mail.present? && !CONFIG['bugzilla_host'].nil?) || User.session
+    - if bugowners_mail.present? && !CONFIG['bugzilla_host'].nil?
+      = render partial: 'webui/project/responsive_ux/actions/report_bug', locals: { bugowners_mail: bugowners_mail, project: project }
+    - if project.is_locked? && User.possibly_nobody.can_modify?(project, true)
+      = render partial: 'webui/project/responsive_ux/actions/unlock_project', locals: { project: project }
+    - elsif User.possibly_nobody.can_modify?(project)
+      = render partial: 'webui/project/responsive_ux/actions/modify_project', locals: { project: project,
+                                                                          packages: packages,
+                                                                          has_patchinfo: has_patchinfo,
+                                                                          open_release_requests: open_release_requests,
+                                                                          release_targets: release_targets }
+      = render partial: 'webui/project/responsive_ux/actions/edit_project', locals: { project: project }
+      = render partial: 'webui/project/responsive_ux/actions/delete_project', locals: { project: project }
+    - elsif !project.is_locked?
+      = render partial: 'webui/project/responsive_ux/actions/request_role_addition_and_deletion', locals: { project: project }
+  - elsif project.key_info.present?
+    = render partial: 'webui/project/responsive_ux/actions/gpg_key_ssl_certificate', locals: { project: project }
+
+  %li.nav-item
+    = link_to(image_templates_path, class: 'nav-link') do
+      %i.fas.fa-compact-disc.fa-lg.mr-2
+      New Image
```

Comment:
@dmarcoux New Image currently appears in the "Actions on this page" section of the actions menu. To me that's more a general action. But not sure if its necessary at this point to introduce an additional area in the action menu for such things. I'm okay with leaving it here

Reply:
I was also a bit divided on this one and in the end, I think it's okay to leave it with the others.